config/vars.yml

```
@@ -1,9 +1,8 @@
-lock_url: 'https://cdn.auth0.com/js/lock/11.6.1/lock.min.js'
+lock_url: 'https://cdn.auth0.com/js/lock/11.13.2/lock.min.js'
```

You can omit the patch version so you get always the latest patch release.

```suggestion
lock_url: 'https://cdn.auth0.com/js/lock/11.13/lock.min.js'
```